px-vis-xy-chart.html

```diff
@@ -607,10 +623,33 @@
         value: false
       },
 
+      /**
+       * Defines the adjusted margin for adding markers. By default the chart
+       * already calculates margins for the markers (taking into account
+       * axes and normal margin) but this can be used to manually adjust
+       * where the markers are drawn
+       * margin object:
+       * - top
+       * - bottom
+       * - left
+       * - right
+       * Not every property needs to be defined, only the one that have been
+       * defined will be used.
+       */
+      marginMarkersAdjustment: {
+        type: Object,
+        value: {},
```

Comment:
This needs to be a function that returns an empty object, otherwise this property will be shared across all instances of this component.

See here for more detail: https://polymer-library.polymer-project.org/1.0/docs/devguide/properties#configure-values

```suggestion
        value: function() { return {}; }
```

Reply:
Done: Updated changes in new commit.